static/bidder-info/compass.yaml

```diff
@@ -0,0 +1,18 @@
+maintainer:
+  email: "sa-support@brightcom.com"
+capabilities:
```

Comment:
Do you have a GVL ID registered with the IAB? I see vendor 943 listed as "Marfeel Solutions, SL (Compass)".

Reply:
Not 943, Client’s Vendor ID is 883

Reply:
Please include id 883 in this configuration file. See https://docs.prebid.org/prebid-server/developers/add-new-bidder-go.html#bidder-info for details.